samples/snippets/requirements.txt

```diff
@@ -0,0 +1,9 @@
+db-dtypes==1.2.0
+google.cloud.bigquery==3.19.0
+google-cloud-bigquery-storage==2.24.0
+ipython===7.31.1; python_version == '3.7'
+ipython===8.0.1; python_version == '3.8'
+ipython===8.18.1; python_version >= '3.9'
+pandas===1.3.5; python_version == '3.7'
+pandas===2.0.3; python_version == '3.8'
+pandas==2.2.1; python_version >= '3.9'
```

Comment:
Please keep this file empty / just add a todo to add `bigquery-magics` after our first release. Also add this comment from https://github.com/googleapis/python-bigquery/blob/main/samples/snippets/requirements.txt#L1

```suggestion
# samples/snippets should be runnable with no "extras"
# TODO(tswast): add bigquery-magics=0.0.01 after first release.
```